README.md

```diff
@@ -112,5 +114,15 @@ If you have any questions or issues, please check this project's [Q&A section in
 ## Discord community & X
 We invite you to join our growing [Discord community](https://www.hanko.io/community) if you want to get the latest updates on passkeys, WebAuthn, and this project or if you just want to chat with us. You can also [follow us on Twitter](https://x.com/hanko_io).
 
+<p align="center">
+    <a href="https://www.hanko.io/community">
+        <img src="https://img.shields.io/static/v1?label=Join&message=Discord&color=blueviolet&style=flat-square" alt="Join Discord">
+    </a>
+    <a href="https://x.com/hanko_io">
+        <img src="https://img.shields.io/twitter/follow/hanko_io?style=social" alt="Follow us on Twitter">
+    </a>
+</p>
```

Comment:
I'd prefer left-aligned buttons (like the cal.com button above)

Reply:
all issue resolved mentioned by you,thankyou